### ISSUE TITLE
Add cropFrac for Lmon

### ISSUE THREAD

**Issue by aliasaadi** [NorESMhub/noresm2cmor#299]
### The Following fields are now supported:
Fields in monthly-averaged output (clm.h0)
```
'cropFrac      ','PCT_LANDUNIT(:,2, :, :)
```

### Sample cmorized fields
under /nird/projects/NS9560K/noresm/cases/NSSP585frc2_f19_tn14_20191014/lnd/hist
under /nird/projects/NS9560K/noresm/cases/NSSP534frc2_f19_tn14_20210427/lnd/hist
```
NSSP585frc2_f19_tn14_20191014.clm2.h0.2015-02-01-00000.nc
NSSP534frc2_f19_tn14_20210427.clm2.h0.2040-01.nc
```

**Comment by YanchunHe** [comment]
I see in CMIP6_Lmon.json file (the control vocabulary file):
```
"cropFrac": {
 "dimensions": "longitude latitude time typecrop”,
```
In the original model output:
```
        double PCT_LANDUNIT(time, ltype, lat, lon) ;
                PCT_LANDUNIT:long_name = "% of each landunit on grid cell" ;
                PCT_LANDUNIT:units = "%" ;
                PCT_LANDUNIT:cell_methods = "time: mean" ;
                PCT_LANDUNIT:_FillValue = 1.e+36 ;
                PCT_LANDUNIT:missing_value = 1.e+36 ;
```
where ltype=9

and in the global attributes:
```
                :ltype_vegetated_or_bare_soil = 1 ;
                :ltype_crop = 2 ;
                :ltype_UNUSED = 3 ;
                :ltype_landice_multiple_elevation_classes = 4 ;
                :ltype_deep_lake = 5 ;
                :ltype_wetland = 6 ;
                :ltype_urban_tbd = 7 ;
                :ltype_urban_hd = 8 ;
                :ltype_urban_md = 9 ;
```

in the google sheet, it also provides `PCT_CFT` as input. I don't know how this variable can be used?

**Comment by aliasaadi** [comment]
As far as I understand, the "PCT_CFT" on the google sheet is not the correct variable to be used for this. PCT_LANDUNIT(time, ltype, lat, lon) with ltype=2 is the right one, which then similar to other cmip6 models will be cmorized as cropFrac(time, lat, lon). 

I see the initials of "YF" in front of this variable on the google sheet. Would be good to add him/her here.

**Comment by YanchunHe** [comment]
> As far as I understand, the "PCT_CFT" on the google sheet is not the correct variable to be used for this. PCT_LANDUNIT(time, ltype, lat, lon) with ltype=2 should be the right one, which then similar to other cmip6 models will be cmorized as cropFrac(time, lat, lon).
> 
> I see the initials of "YF" in front of this variable on the google sheet. Would be good to add him/her here.

I updated the corresponding google sheet entry and will try this.

"YF" has left the work here, so not need to include him.

**Comment by aliasaadi** [comment]
Great, thank you very much!

**Comment by YanchunHe** [comment]
fixed in commit 9b4b7db
datasets newer from v20210811 will include `cropFrac` in `Lmon`.

close this issue now. reopen if needed.
